10-std.tex

```diff
@@ -368,6 +368,98 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
+\section{Template}
+\label{std-template}
+
+Haxe comes up with a standard Template System with an easy to use syntax which is interpreted by a lightweight class called \type{haxe.Template}.
+
+A Template is a string or a file that is used to produce some HTML or XML output depending on the input. Here is a small template example:
+
+\haxe{assets/Template.hx}
+
+The console will trace \ic{My name is Mark, 30 years old}.
+
+\paragraph{Expressions}
+Expression can be put between the \ic{::}, and the syntax allows the current possibilities:
```

Comment:
"Expressions" or "an expression", no comma before and. Delete the and, keep the comma